your-first-cluster.md

```diff
@@ -521,3 +522,7 @@ kubectl --kubeconfig cluster-one.kubeconfig get nodes
 
 $env:NO_COLOR = 'true'; clusterctl describe cluster docker-cluster-one
 ```
+
+## Changing Cluster Topology
+
+Now we're ready to dive into the power of Cluster API, [let's explore cluster topology](cluster-topology.md)!
```

Comment:
Can you update/move the "chaining" comments?

The doc order until this PR is:
* Prereqs
* Creating your first cluster with Cluster API
* Deploying the Cluster API visualizer
* Changing the Cluster Topology
* MachineHealthChecks and remediation

So after first cluster we have the visualizer then changing the cluster topology.

If you have time, would be great if you can add similar comments to prereqs, at the beginning of your first cluster, ...

Thank you very much!

Reply:
done